awx/ui/client/src/shared/stateDefinitions.factory.js

```diff
@@ -848,7 +848,13 @@ function($injector, $stateExtender, $log, i18n) {
                                 // Need to change the role_level here b/c organizations and inventory scripts
                                 // don't have a "use_role", only "admin_role" and "read_role"
                                 if(list.iterator === "organization"){
-                                    $stateParams[`${list.iterator}_search`].role_level = "admin_role";
+                                    if ($state.current.name.includes('inventories')) {
```

Comment:
The only thing I'll mention here is that we should be completely certain that `$state.current.name` will _always_ be defined once this part of the code is reached. Feel free to ignore this comment if you've already double-checked.